web/static/js/components/idea_submission_form.jsx

```diff
@@ -101,7 +101,7 @@ export class IdeaSubmissionForm extends Component {
       pointerText = !ideaEntryStarted ? "Create Action Items!" : ""
       dropdownProps = {
         labelName: "assignee",
-        value: assigneeId,
+        value: assigneeId || "",
```

Comment:
maybe i'm being paranoid, but does this fallback value throw errors server side? i think we'll want assigneeId to explicitly be passed as `null` when passed to the server in cases where there is no assigneeId

Reply:
It doesn't send the fallback to the server. This is only there to stop console errors resulting from having the initial value from the state set as `null`.